src/index.css

```diff
@@ -68,7 +68,7 @@ header {
     gap: 10px;
 }
 
-#call-delete-wrapper {
+#call-remove-button-wrapper {
```

Comment:
Still not great:
1. It doesn't tell where this div belongs (Keypad view)
2. If there was 10000 of buttons inside, would you name all of them? No
3. I would use `keypad-actions-wrapper`